src/genie/libs/parser/iosxe/tests/ShowWirelessStatsClientDeleteReasons/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,218 @@
+            "static_ip_anchor_discovery_failure": 0,
+            "vlan_failure": 0,
+            "acl_failure": 0,
+            "redirect_acl_failure": 11,
+            "accounting_failure": 0,
+            "security_group_tag_failure": 0,
+            "fqdn_filter_definition_does_not_exist": 0,
+            "wrong_filter_type,_expected_postauth_fqdn_filter": 0,
+            "wrong_filter_type,_expected_preauth_fqdn_filter": 0,
+            "invalid_group_id_for_fqdn_filter_valid_range__1_16": 0,
+            "policy_parameter_mismatch": 0,
+            "reauth_failure": 0,
+            "wrong_psk": 0,
+            "policy_failure": 0,
+            "aaa_server_unavailable": 0,
+            "aaa_server_not_ready": 0,
+            "no_dot1x_method_configuration": 0,
+            "association_connection_timeout": 0,
+            "mac_auth_connection_timeout": 0,
+            "l2_auth_connection_timeout": 1752,
+            "l3_auth_connection_timeout": 9584,
+            "mobility_connection_timeout": 0,
+            "static_ip_connection_timeout": 0,
+            "sm_session_creation_timeout": 0,
+            "ip_learn_connection_timeout": 46021,
+            "nack_ifid_exists": 0,
+            "guest_lan_invalid_mbssid": 0,
+            "guest_lan_no_memory": 0,
+            "guest_lan_ceate_request_failed": 0,
+            "eogre_reset": 0,
+            "eogre_generic_join_failure": 0,
+            "eogre_ha_reconciliation": 0,
+            "wired_idle_timeout": 0,
+            "ip_update_timeout": 0,
+            "sae_commit_received_in_associated_state": 0,
+            "nack_ifid_mismatch": 0,
+            "eogre_invalid_vlan": 0,
+            "eogre_empty_domain": 0,
+            "eogre_invalid_domain": 0,
+            "eogre_domain_shut": 0,
+            "eogre_invalid_gateway": 0,
+            "eogre_all_gateways_down": 0,
+            "eogre_flex___no_active_gateway": 0,
+            "eogre_rule_matching_error": 0,
+            "eogre_aaa_override_error": 0,
+            "eogre_client_onboarding_error": 0,
+            "eogre_mobility_handoff_error": 0,
+            "l3_vlan_override_connection_timeout": 0,
+            "delete_received_from_ap": 0,
+            "qos_failure": 0,
+            "wpa_group_key_update_timeout": 318,
+            "client_blacklist": 0,
+            "dot11_unsupported_client_capabilities": 0,
+            "dot11_association_denied_unspecified": 0,
+            "dot11_ap_have_insufficient_bandwidth": 0,
+            "dot11_invalid_qos_parameter": 0,
+            "client_not_allowed_by_assisted_roaming": 0,
+            "wired_client_deleted_due_to_wgb_delete": 0,
+            "client_abort": 0,
+            "mobility_peer_delete": 0,
+            "no_ip": 469,
+            "bssid_down": 1,
+            "dot11_qos_policy": 0,
+            "roam_across_policy_profile_deny": 0,
+            "4way_handshake_failure___m1_issue": 0,
+            "4way_handshake_failure___m3_issue": 0,
```

Comment:
`4way_handshake_failure___m1_issue` to `4way_handshake_failure_m1_issue` (1x underscore)

Reply:
Updated